v2/util/time.go

```diff
@@ -57,6 +57,7 @@ var (
 	MozillaPolicy22Date         = time.Date(2013, time.July, 26, 0, 0, 0, 0, time.UTC)
 	MozillaPolicy24Date         = time.Date(2017, time.February, 28, 0, 0, 0, 0, time.UTC)
 	MozillaPolicy27Date         = time.Date(2020, time.January, 1, 0, 0, 0, 0, time.UTC)
+	CABFBRs_1_6_9_Date          = time.Date(2020, time.March, 27, 0, 0, 0, 0, time.UTC)
```

Comment:
One thing I debated: What to name this. The naming here for CAB related things is a bit weird. Up until CABV201Date, it *seems* like the naming for CABV actually tied to the version of the document (in which case, this should be `CABV169Date`

Except 201 was a ballot number, not a version number, since the EV Guidelines and the BRs are separately versioned.

I opted here to be explicit about document-and-version, in line with the ETSI date, but I could also see an argument for a descriptive date, like `OnionV3Date`, so that folks don't have to constantly figure out what changed in BRs 1.6.9 vs 1.7.0.

Alternatively, naming the Ballots, e.g. as `CABFSC27`, at least makes it easier to quickly look up what the diff was (and across documents).

Naming is hard, but I wanted to draw attention to this to see where folks felt.

Reply:
> Except 201 was a ballot number, not a version number, since the EV Guidelines and the BRs are separately versioned.

I think this one was my fault :sweat: In retrospect I agree it seems better to match to the document version number instead of referencing a ballot here.

> I opted here to be explicit about document-and-version,

That feels right for this case to me too :+1: